### ISSUE TITLE
Add configurable keybinds for the web interface

### ISSUE THREAD

**Issue by mvrozanti** [diaspora/diaspora#8022]
Hi there 
I love this project
I was wondering if configurable keybinds could be a wanted feature. I noticed 'k' and 'j' are already bound (thanks random vim user!) for the stream and 'l', 'c', 'o' and 'r' for a single post, but it would be nice to have the ability to change those and add some others

**Comment by Amorn-Kingthong** [comment]
i will try

**Comment by denschub** [comment]
@Amorn-Kingthong please open a thread on Discourse first. This actually is a potentially controversial feature request, since we usually try to keep the number of settings as low as possible to avoid user confusion, so we really should have a thorough discussion here before starting any work.

**Comment by mvrozanti** [comment]
While I understand why that may be the case for some users, simply "hiding" this option from the main pages should be sufficient to minimize "confusion". Default values, like the ones in place, should be the "hint" to the user that there are keystroke shortcuts and if the user so desires to change them, he'll go on his way to find out where these are set, possibly through a manual page.

**Comment by denschub** [comment]
I disagree with adding settings for everything for multiple reasons, and that's why I suggested having that discussion on the place where it belongs.

**Comment by mvrozanti** [comment]
I think there's a topic on there already:

https://discourse.diasporafoundation.org/t/add-change-keyboard-shortcuts/293/17
